---
geometry_msgs/PoseStamped[] bin_poses